<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgrammersClass</class>
 <widget class="QWidget" name="ProgrammersClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Programmers</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout"/>
     </item>
     <item>
      <widget class="QListWidget" name="listTasks"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Task description:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Description"/>
     </item>
     <item>
      <widget class="QPushButton" name="AddBut">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeBut">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startBut">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="closeBut">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="programmers.qrc"/>
 </resources>
 <connections/>
</ui>
